features/pom.xml

```diff
@@ -16,6 +16,17 @@
 
   <name>openHAB Add-ons :: Features :: Karaf</name>
 
+  <properties>
+    <!-- JDBC database driver versions -->
```

Comment:
@kaikreuzer Can you briefly if there was a special reason for adding the JDBC version properties in this file? It created confusion in https://github.com/openhab/openhab-addons/pull/9810#issuecomment-760533170.

Can we remove them? Or add them in the in the feature file of the bundle?

Reply:
I don't recall, sorry. If it works without it, we can remove them.

Reply:
The code compiles successfully when I remove the properties from this file. I did not test installing the features.